app/controllers/shoutouts_controller.rb

```diff
@@ -32,7 +32,8 @@ def create
       end
     end
 
-    if params[:text].include?(params[:user_name])
+    user_name = params[:user_name]
+    if (params[:text].include?('@'+user_name+' ') || (params[:text][-(user_name.size)..params[:text].size].eql?(user_name)))
```

Comment:
I don't really understand what the 2nd part of the conditional is doing that the first part doesn't cover?

The first part has some obvious issues in that I could probably cheer for `@myself, I am great` - does the 2nd part block that?

I wonder if we can take some inspiration from below when we create the Shoutout.  Particularly the recipients bit: `message.scan(/(@(\w+|\.)+)/).map(&:first).map { |r| r[1..-1] }`.  This pulls all the `@<whatever>`'s (where `<whatever>` is made up of word characters and `.`s) and creates a list of them without their leading `@`-signs to be the recipients.  Maybe we can do the same thing and then check that list doesn't contain `user_name`.  It'd add some consistency to it, and hopefully avoid the `/cheer @myself, I am great` issue.